freeplane/src/main/java/org/freeplane/view/swing/map/MapViewScrollPane.java

```diff
@@ -85,14 +85,10 @@ public void setBackgroundComponent(JComponent backgroundComponent) {
         public void scrollRectToVisible(Rectangle newContentRectangle) {
         	final Rectangle hiddenArea = hiddenAreaSupplier.getHiddenArea();
         	if(hiddenArea.width != 0 && hiddenArea.height != 0) {
-        		Point viewportLocation = new Point(0, 0);
-        		UITools.convertPointToAncestor(this, viewportLocation, JScrollPane.class);
-        		hiddenArea.x -= viewportLocation.x;
-        		hiddenArea.y -= viewportLocation.y;
         		final boolean isHiddenAreaAtTheLeft = hiddenArea.x == 0;
 				final boolean isHiddenAreaAtTheTop = hiddenArea.y == 0;
-				final boolean isHiddenAreaAtTheRight = hiddenArea.x + hiddenArea.width == getWidth();
-				final boolean isHiddenAreaAtTheBottom = hiddenArea.y + hiddenArea.height == getHeight();
+				final boolean isHiddenAreaAtTheRight = hiddenArea.x + hiddenArea.width == getParent().getWidth();
```

Comment:
I do not understand why this change was needed. Could you explain?

Reply:
Please take a look at the latest commit.